### PR TITLE
IGAPP-394: Add proper a11y label for kebab menu

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

Check bug description in[ IGAPP-394](https://issues.tuerantuer.org/browse/IGAPP-394) for testing on android real device

### CODE REVIEW

**Review by SteffiStuffel — APPROVED** [review]
(opional): Add release note?
not tested.

File: translations/translations.json [hunk 1]
```diff
@@ -349,6 +350,7 @@
       "loading": "Φόρτωση ..."
     },
     "en": {
+      "moreOptions": "More Options",
```
Comment (klinzo) [inline translations/translations.json:353]: Only german translations should be added
```suggestion
```

File: translations/translations.json [hunk 2]
```diff
@@ -313,6 +313,7 @@
   },
   "common": {
     "de": {
+      "moreOptions": "Weitere Optionen",
```
Comment (klinzo) [inline translations/translations.json:316]: Did you have a look what accessibility label other apps are setting? Not sure if this is a good one.

Comment (f1sh1918) [reply translations/translations.json:316]: Mattermost has "Mehr Optionen" if you prefer that :D

Comment (klinzo) [reply translations/translations.json]: Both is fine :) Mehr is perhaps a little bit simpler language but your choice :)

**Review by klinzo — APPROVED** [review]

**Pull request merged**